From a86ad5af001c7ccfd3a145c694e1a14a5514b77d Mon Sep 17 00:00:00 2001
From: NamNguyen <nam.nguyen.xb@rvc.renesas.com>
Date: Tue, 15 Nov 2016 15:37:38 +0700
Subject: [PATCH 3/3] adding flexible display for video playback demo Note:
 video playback demo use RZ_GST-demo_video-playback.sh to play video so you
 have to make sure that RZ_GST-demo_video-playback.sh was added scaling
 feature.

---
 hmi-scripts/Video_Playback.sh | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/hmi-scripts/Video_Playback.sh b/hmi-scripts/Video_Playback.sh
index 90a572c..d766381 100644
--- a/hmi-scripts/Video_Playback.sh
+++ b/hmi-scripts/Video_Playback.sh
@@ -1,4 +1,5 @@
 #!/bin/bash
+source /home/root/.bashrc # get D_WIDTH, D_HEIGHT
 
 TTY=$(grep "\/sbin\/getty" /etc/inittab | grep 'ttyS' | awk '{ print $NF }')
 TTY=/dev/${TTY}
@@ -9,4 +10,4 @@ printf "\tYou are watching Video playback on screen\n\
 \t(/home/root/videos/h264-fhd-60.mp4)\n" > ${TTY}
 printf "\n\e[31mNOTE: Click on Exit icon to exit from running applications\e[0m\n\n" > ${TTY}
 
-/home/root/RZ_scripts/RZ_GST-demo_video-playback.sh /home/root/videos/h264-fhd-60.mp4 > ${TTY}
+/home/root/RZ_scripts/RZ_GST-demo_video-playback.sh /home/root/videos/h264-fhd-60.mp4 0 0 $D_WIDTH $D_HEIGHT > ${TTY}
-- 
1.9.1

